### ISSUE TITLE
Safe to start Ticker timer from interrupt

### ISSUE THREAD

**Issue by supersjimmie** [esp8266/Arduino#2511]
#### Hardware
Hardware:			NodeMCU v2
Core Version:      	2.3.0

### Description

I have an CC1101 RF receiver that can trigger an interrupt when data comes in. After that interrupt it will take roughly 10ms for all to buffer and I need the buffer to be filled before handling it.
My idea is to attach a 10ms ticker within the interrupt handler and handling the rf buffer from the ticker. After that I need to detach the ticker from within itself.

### Stripped Sketch

```cpp

#define INT_PIN D3

#include <Ticker.h>
Ticker ITHOticker;

void ITHOcheck() {
// READ THE DATA INTO MY VARS HERE
  ITHOticker.detach();
}

void ICACHE_RAM_ATTR ITHOinterrupt() {
  ITHOticker.attach_ms(10, ITHOcheck);
}

void setup(void) {
  Serial.begin(115200);
  pinMode(INT_PIN, INPUT);
  attachInterrupt(INT_PIN, ITHOinterrupt, RISING);
}

void loop(void) {
// DO SOMETHING WITH THE VARS HERE
}
```

So what I basically would like to know, is if it is safe to:
1. Attach a ticker from the interrupt function
2.  Detach the ticker from within the ticker function itself.

**Comment by supersjimmie** [comment]
ah, about detaching it: I found that I can also use
```ITHOticker.once_ms(10, ITHOcheck);
```
So now I only need to be sure that it is safe to start a ticker from an ISR.

**Comment by supersjimmie** [comment]
I've done some testing and it just seems to work.
I don't know if my solution is wise and stable but since it just works I will close this.
